File: API/Features/Sales/Prices/ViewModels/PriceListVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Sales.Prices {

    public class PriceListVM {

        public int Id { get; set; }
        public SimpleEntity HullType { get; set; }
        public SimpleEntity SeasonType { get; set; }
        public string Code { get; set; }
        public string Description { get; set; }
        public string EnglishDescription { get; set; }
        public bool IsIndividual { get; set; }
        public int Length { get; set; }
        public decimal NetAmount { get; set; }
        public decimal VatPercent { get; set; }
        public decimal VatAmount { get; set; }
        public decimal GrossAmount { get; set; }

    }

}

File: FrontEnd/src/app/features/sales/prices/classes/view-models/price-list-vm.ts
import { SimpleEntity } from "src/app/shared/classes/simple-entity"

export interface PriceListVM {

    id: number
    hullType: SimpleEntity
    periodType: SimpleEntity
    seasonType: SimpleEntity
    code: string
    description: string
    englishDescription: string
    isIndividual: boolean
    length: number
    netAmount: number
    vatAmount: number
    grossAmount: number

}
